m4_sessions/1810-inning/creativity.md

```diff
@@ -1,25 +1,35 @@
 ## Group Members:
 
+Dennis Miller, Laura Whitaker, Jesse McFadden, Paul Kim
 
 
 ## Name of Session: 
 
+Create and Engineer
+
 ## Session Description (Why should students come to this session): (EX: Are you interested in biological enhancement? Drop in to learn about biohacking.)
 
+
 ## Session Outcomes 
 
 (What will students learn/leave with?): 
 
-
+1) Feeling inspired and energized
+2)Takes them out of their element 
+3) Having a next step.
+4)feeling like they made a connection
 
      Best practices:  No section should be longer than 20 minutes
      Create a session that YOU would want to attend
      Share resources in your slack channel
      All sessions NEED to have a closing activity, reflection, discussion
-## Session Outline 
+## Session Outline
 
-(Please include who is facilitating and how much time you are going to spend in each section):
+     Ice Breaker: What is one creative thing you did this week?
+     
+0) Ice Breaker: What is one creative thing you did this week? 1) Inspired/Energized  2) Have done something that made them uncomfortable(feeling good about it)  3) Feeling like they made a connection. (edited)
 
+1)Exquisite corpse. 2) Ted talk on creativity(or something) 3) Brainstorming/Discussion
 ## Recommended Trajectory 
 
```

Comment:
Think about how to make this connected to the day to day...how can your creativity help you with problem solving, project options, future projects?